layouts/shortcodes/versions.html

```diff
@@ -0,0 +1,3 @@
+<div class="versions-border">
```

Comment:
Do you think it would be nice to make this look a bit more like the download buttons? (Like I did for the current version of the webpage)

Reply:
Don't think it will look good now due to the `details` tag.

Reply:
I tried to fit it in, by making it look like the search field.

Reply:
Meeting after 2 weeks since I've been really busy. As requested, the changes have been made in https://github.com/neoforged/websites/pull/15/commits/04be4272934440318ca5847a1c39477a4d8de1dd.